homeassistant/components/bthome/sensor.py

```diff
@@ -316,7 +315,7 @@
         key=f"{BTHomeSensorDeviceClass.VOLUME}_{Units.VOLUME_LITERS}",
```

Comment:
Since volume can be both a "total" (amount of water which has flowed through a pipe) and a measurement (amount of water in a gas tank), please add comments explaining which one this is. Or could it be both since bthome allows to create your own sensors?

Reply:
I checked the bthome documentation (https://bthome.io/format/), I think this should be the stored amount of volume, as the volume flow rate is below. So we need to keep the state class and change the device class to VOLUME_STORAGE.

Reply:
volume flow rate is in m3/hr, that is something different. 

I think we have to create two different sensors in https://github.com/Bluetooth-Devices/bthome-ble

A `VOLUME_STORAGE` sensor with and SensorStateClass `measurement`
A `VOLUME` sensor with UoM Liter and SensorStateClass `total`

Reply:
My proposal would be that you use SensorStateClass `total` for the current `VOLUME` sensor. The `VOLUME_STORAGE` sensor can be added later in a different PR. I have made an issue on the bthome repo here for that request. https://github.com/Bluetooth-Devices/bthome-ble/issues/100

Reply:
Ok perfect, thanks for creating the issue in the bthome repo. Than this PR is fine. I spoke to @emontnemery about the LTS and that would be fine as the LTS have been wrong since the beginning.

Reply:
I have added the `VOLUME STORAGE` sensor to bthome 3.4.1

https://github.com/Bluetooth-Devices/bthome-ble/releases/tag/v3.4.0
https://github.com/Bluetooth-Devices/bthome-ble/releases/tag/v3.4.1

Will add it in a different PR to HA.